patches/chromium/refactor_unfilter_unresponsive_events.patch

```diff
@@ -0,0 +1,47 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Samuel Maddock <smaddock@slack-corp.com>
+Date: Tue, 12 Nov 2024 17:30:42 -0500
+Subject: refactor: unfilter unresponsive events
+
+RendererUnresponsive events are filtered out when
```

Comment:
What's the path forward for this patch? Do you intend it be floated indefinitely?

Reply:
I think it's okay to float it indefinitely, since we handle these errors fundamentally differently than Chrome - it would be nice to upstream a suggestion to let us customize this a bit, but I don't see an immediate path forward there

Reply:
I'll try to upstream what I can https://chromium-review.googlesource.com/c/chromium/src/+/6020347